### DISCUSSION TITLE
Integration of the STOP command

### DISCUSSION THREAD

**Discussion by Gotouille** [clementprevot/home-assistant-idiamant#62] [Ideas]
Hi Clément,
Are you planning to integrate the STOP command that is missing from the current main integration of idiamant in HAss?

Do you have an idea of a timeframe that you need in order to make the integration work?

I'm sorry that I am no help for you.

**Comment by clementprevot** [comment]
Hello!

Yes the stop command will be included (in fact it's as simple as sending a "-1" position to the store to stop it)
I wasn't aware of another integration of iDiamant. Maybe it would be better to implement the stop command instead of creating a new integration.

Moreover as I'm not sure when I will be able to finish it.

**Comment by Gotouille** [comment]
Hi,
Thank you for your answer.
Well, the only integration I know of, the one I mention, is not a proper
one. Currently the idiamant integration is made through the homekit
controller integration. So I don't know if this is that easy to modify.
What do you think?

Regards

**Reply by clementprevot** [reply]
Ah OK, if it's a Homekit integration it's not ideal and I'm not even sure the -1 value trick will work.
My integration will use the direct API.
